src/test/scala/it/pagopa/interop/catalogprocess/CatalogProcessSpec.scala

```diff
@@ -669,6 +669,102 @@ class CatalogProcessSpec extends SpecHelper with AnyWordSpecLike with ScalatestR
         status shouldEqual StatusCodes.OK
       }
     }
+    "succeed when technology has changed" in {
+      val requesterId = UUID.randomUUID()
+
+      implicit val context: Seq[(String, String)] =
+        Seq("bearer" -> bearerToken, USER_ROLES -> "admin", ORGANIZATION_ID_CLAIM -> requesterId.toString)
+
+      val catalogDescriptor =
+        SpecData.catalogDescriptor.copy(state = Draft, interface = Some(SpecData.catalogDocumentInterface))
+
+      val descriptor =
+        SpecData.eServiceDescriptor.copy(
+          state = CatalogManagementDependency.EServiceDescriptorState.DRAFT,
+          interface = None
+        )
+
+      val eService = SpecData.catalogItem.copy(descriptors = Seq(catalogDescriptor), producerId = requesterId)
```

Comment:
For clarity and to enforce data preparation
```suggestion
      val eService = SpecData.catalogItem.copy(technology = EServiceTechnology.REST, descriptors = Seq(catalogDescriptor), producerId = requesterId)
```